### PR TITLE
Set connect timeout to MixPanel API endpoint

### LINKED ISSUE #20: Connect timeout not set
#12 only fixated a read timeout. However, a sane Java library should also set connect timeout, which defaults to infinity. Without setting it, the client risks locking up an entire application that has fixed number of threads (which most do).

See also 9f945f39423de7408d4ad220cdada0721b383607.

### PR DESCRIPTION
**Pull request by JensRantil**
Fixes #20.

### CODE REVIEW

File: src/main/java/com/mixpanel/mixpanelapi/MixpanelAPI.java [hunk 1]
```diff
@@ -200,5 +201,6 @@ private String slurp(InputStream in) throws IOException {
 
     private static final int BUFFER_SIZE = 256; // Small, we expect small responses.
     private static final int READ_TIMEOUT_MILLIS = 120000; // Two minutes should be more than enough for a response.
+    private static final int CONNECT_TIMEOUT_MILLIS = 30000; // Two minutes should be more than enough for a response.
```
Comment (avram) [inline src/main/java/com/mixpanel/mixpanelapi/MixpanelAPI.java]: This is 30 seconds.

Comment (JensRantil) [inline src/main/java/com/mixpanel/mixpanelapi/MixpanelAPI.java]: Sloppy of me. Fixed in 45c23720bfd148fa6e4912ab315d9b69a0fe5196. Thanks for feedback.

**Comment by JensRantil** [comment]
Hello. Any input on this?

**Comment by patedit** [comment]
@JensRantil Sorry for the late response. Merging :+1:

**Pull request merged**